packages/venia-ui/lib/components/Navigation/navHeader.js

```diff
@@ -21,34 +18,42 @@ const titles = {
 };
 
 const NavHeader = props => {
-    const { isTopLevel, onBack, onClose, view } = props;
+    const { isTopLevel, onBack, view } = props;
 
     const talonProps = useNavigationHeader({
         isTopLevel,
         onBack,
-        onClose,
         view
     });
 
-    const { handleClose, handleBack, isTopLevelMenu } = talonProps;
-    const title = titles[view] || titles.MENU;
+    const { handleBack, isTopLevelMenu } = talonProps;
+
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const titleClassName = ['MY_ACCOUNT', 'SIGN_IN'].includes(view)
```

Comment:
This looks like a weird use case to only capitalize those 2 titles.

Reply:
Get rid of uppercase in general and use `capitalize` everywhere

Reply:
Resolved by https://github.com/magento/pwa-studio/commit/d75dd9ceedd7e609fe60e01358f36851b6097c89.